mne_neuron/tests/test_compare_hnn.py

```diff
@@ -0,0 +1,33 @@
+import os.path as op
+
+from numpy import loadtxt
+from numpy.testing import assert_array_equal
+
+from mne.utils import _fetch_file
+import mne_neuron
+from mne_neuron import simulate_dipole, Params, Network
+
+
+def test_mne_neuron():
+    """Test to check if MNE neuron does not break."""
+    # small snippet of data on data branch for now. To be deleted
+    # later. Data branch should have only commit so it does not
+    # pollute the history.
+    data_url = ('https://raw.githubusercontent.com/jasmainak/'
+                'mne-neuron/test_data/dpl.txt')
```

Comment:
@blakecaldwell would you mind providing me the dipole text file against which you are testing? I want to add it to the tests so that it doesn't hinder development of mne-neuron once it's fixed. I don't think what I have currently is the correct one ...